### ISSUE TITLE
Audio connection fail using >=Chrome-96

### ISSUE THREAD

**Issue by gjaekel** [igniterealtime/openfire-pade-plugin#380]
It was reported to me, that Audio fails in Conferences with recent versions of the Chrome browser (but not on Chromium).

In the browsers dev console, you'll find
```
lib-jitsi-meet.min.js:9910 2022-01-10T11:09:04.220Z [JitsiMeetJS.js] <Object.getGlobalOnErrorHandler>:  UnhandledError: Failed to construct 'RTCPeerConnection': Plan B SDP semantics is a legacy version of the Session Description Protocol that has severe compatibility issues on modern browsers and is no longer supported. See https://www.chromestatus.com/feature/5823036655665152 for more details, including the possibility of registering for a Deprecation Trial in order to extend the Plan B deprecation deadline for a limited amount of time.
```

https://chromestatus.com/feature/5823036655665152#details says:

### Feature: [WebRTC] Deprecate and Remove Plan B

The SDP used to establish a connection in WebRTC has a non-standard dialect: Plan B. Removal timeline:
M93: Exception thrown in Canary.
M96: Exception thrown in Beta and Stable.
There is a Deprecation Trial to avoid the exception until May 25, 2022 (https://developer.chrome.com/origintrials/#/view_trial/3892235977954951169). The End Date was previously December 29. To get new new End Date, a new OT token is needed.
First version with Plan B removed: M102.

**Comment by deleolajide** [comment]
I am not experiencing this with latest version 1.5.2 which is using latest Jitsi code. Could you please confirm what version of pade plugin that this is happening. Thanks

**Comment by gjaekel** [comment]
Indeed, this was reported for the old (but our current) version `pade.v1_4_1.20210606-18210`. I'll try to check out the latest release ASAP.

**Comment by gjaekel** [comment]
Fixed with V1.6
